soh/soh/Enhancements/randomizer/3drando/fill.cpp

```diff
@@ -22,6 +22,57 @@
 using namespace CustomMessages;
 using namespace Rando;
 
+
+//RANDOTODO merge into Logic once Logic is a class passed to logic funtions 
+struct GetAccessableLocationsStruct {
+  std::vector<RandomizerCheck> accessibleLocations;
+  std::vector<RandomizerRegion> regionPool;
+  //Variables for playthrough
+  int gsCount;
+  int maxGsCount;
+  std::vector<LogicVal> buyIgnores;
+
+  //Variables for search
+  std::vector<Rando::ItemLocation*> newItemLocations;
+  bool updatedEvents;
+  bool ageTimePropogated;
+  bool firstIteration;
+
+  //Variables For Validating Entrences
+  bool haveTimeAccess;
+  bool foundTempleofTime;
+  bool validatedStartingRegion;
+  bool sphereZeroComplete;
+
+  std::vector<RandomizerCheck> itemSphere;
+  std::list<Entrance*> entranceSphere;
+
+  GetAccessableLocationsStruct(int _maxGsCount){
+    regionPool = {RR_ROOT};
+    gsCount = 0;
+    maxGsCount = _maxGsCount;
+    updatedEvents = false;
+    ageTimePropogated = false;
+    firstIteration = true;
+    haveTimeAccess = false;
+    foundTempleofTime = false;
```

Comment:
I know it's weird because usually of isn't capitalized in text, but I'd like to still have it be `foundTempleOfTime` for ease of reading.